src/drive/web/modules/views/Drive/index.jsx

```diff
@@ -1,50 +1,51 @@
-/* global __TARGET__ */
 import React, { useCallback, useContext } from 'react'
 import { connect } from 'react-redux'
 import { useQuery } from 'cozy-client'
+import get from 'lodash/get'
+import uniqBy from 'lodash/uniqBy'
 
-import SharingProvider, { SharingContext } from 'cozy-sharing'
-import {
-  ThumbnailSizeContext,
-  ThumbnailSizeContextProvider
-} from 'drive/lib/ThumbnailSizeContext'
-import { ModalStack, ModalContextProvider } from 'drive/lib/ModalContext'
-import { RouterContextProvider } from 'drive/lib/RouterContext'
+import { SharingContext } from 'cozy-sharing'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import useActions from 'drive/web/modules/actions/useActions'
 
-import SelectionBar from 'drive/web/modules/selection/SelectionBar'
-import Dropzone from 'drive/web/modules/upload/Dropzone'
-import Main from 'drive/web/modules/layout/Main'
-import Topbar from 'drive/web/modules/layout/Topbar'
+import FolderView from '../Folder/FolderView'
+import FolderViewHeader from '../Folder/FolderViewHeader'
+import FolderViewBody from '../Folder/FolderViewBody'
+import FolderViewBreadcrumb from '../Folder/FolderViewBreadcrumb'
 import Toolbar from 'drive/web/modules/drive/Toolbar'
 import { ROOT_DIR_ID } from 'drive/constants/config'
 
-import { FileListv2 } from 'drive/web/modules/filelist/FileList'
-import { ConnectedFileListBodyV2 as FileListBodyV2 } from 'drive/web/modules/filelist/FileListBody'
-import AddFolder from 'drive/web/modules/filelist/AddFolder'
-import FileListHeader from 'drive/web/modules/filelist/FileListHeader'
-import MobileFileListHeader from 'drive/web/modules/filelist/MobileFileListHeader'
-import Oops from 'components/Error/Oops'
-import { EmptyDrive } from 'components/Error/Empty'
-import FileListRowsPlaceholder from 'drive/web/modules/filelist/FileListRowsPlaceholder'
-import { isMobileApp } from 'cozy-device-helper'
-import LoadMore from 'drive/web/modules/filelist/LoadMoreV2'
-import Breadcrumb from './Breadcrumb'
-import { FileWithSelection as File } from 'drive/web/modules/filelist/File'
 import { buildDriveQuery } from 'drive/web/modules/queries'
 import { getCurrentFolderId } from 'drive/web/modules/selectors'
 import { useFolderSort } from 'drive/web/modules/navigation/duck'
-import { ModalManager } from 'react-cozy-helpers'
-
-import useActions from 'drive/web/modules/actions/useActions'
 
-import RealTimeQueries from './RealTimeQueries'
+const getBreadcrumbPath = (t, displayedFolder) =>
+  uniqBy(
+    [
+      {
+        id: ROOT_DIR_ID
+      },
+      {
+        id: get(displayedFolder, 'dir_id')
```

Comment:
Could we add the `name` here ? It'd be easier to know where we are.

Reply:
Ah I see, we do not have the parent path here. Having the parent relationship on `files` would help here I think.